L103-python-add-typing-to-sync-api.md

```diff
@@ -70,9 +63,11 @@ def __call__(self,
 
 ## Implementation
 
-1. Add typing to internal APIs.
-2. Add typing to public API and test though pre-release.
-3. Add typing to public APIs.
+* Add typing to internal APIs.
+  * [PR1: Add typing for some internal python files.](https://github.com/grpc/grpc/pull/31514)
+* Add typing to public API and test though pre-release.
```

Comment:
Let's mark the typing as experimental for at least one release. This will give us space to react to any feedback that people might give us as they adopt it.

Since we can't add the types to the experimental module, this is going to mean docstrings and mentioning that these are currently experimental in the release notes.